runtime/vm/vm_internal.h

```diff
@@ -431,6 +431,18 @@ convertByteArrayToCString(J9VMThread *currentThread, j9object_t byteArray);
 j9object_t
 convertCStringToByteArray(J9VMThread *currentThread, const char *byteArray);
 
+/**
+ * Allocate native memory and copy the argument array to it.
+ *
+ * @param currentThread[in] the current J9VMThread
+ * @param argArray[in] the specified argument array (must not be NULL)
+ * @param javaArgs[in] the specified native memory to store the arguments (must not be NULL)
+ *
+ * @returns the newly-allocated memory, or NULL on failure (no exception is set pending)
+ */
+UDATA *
+convertToNativeArgArray(J9VMThread *currentThread, j9object_t argArray, UDATA *javaArgs);
```

Comment:
I believe these should also be U64.

Reply:
Ditto for the function declaration.

Reply:
Replaced with `U_64`.

Reply:
This is still `UDATA`, not `U_64`. Did I miss some discussion that concluded differently?